src/Sidebar/MenuItem.tsx

```diff
@@ -12,7 +12,7 @@ import { makeStyles } from "../theme";
 import { SidebarMenuItem } from "./types";
 
 export interface MenuItemProps {
-  active: boolean;
+  active: string;
```

Comment:
`active` implies a boolean, since it's compared to an id maybe change the name to `activeId`